### ISSUE TITLE
about compute dnf

### ISSUE THREAD

**Issue by jialei1** [YichiCS/Diffusion-Noise-Feature#10]
Thank you for your work. I reviewed the training code and found that it seems to be independent of the code for calculating dnf. Is it not necessary to calculate dnf during the training process? Is dnf directly processed on the training set and then used for training? In that case, do we also need to process the test set and validation set in advance? Also, I would like to ask about the training set provided by dire, such as lsun_bedroom, which includes a real set and multiple fake sets. Should these fake sets be trained together or should we only select one?Could you please provide guidance?

**Comment by YichiCS** [comment]
Hello，

1) Precomputing images into DNF during training and validation helps reduce unnecessary inference time, as DNF may be used multiple times throughout the process. It is also necessary to convert them to DNF before validation and testing.
2) Please use the entire training set.

**Comment by jialei1** [comment]
thanks for your quick reply
